src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -5,11 +5,16 @@
  */
 public class CliSyntax {
 
-    /* Prefix definitions */
+    /* Person Prefix definitions */
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
 
+    /* Order Prefix definitions */
+    public static final Prefix PREFIX_DESCRIPTION = new Prefix("i/");
```

Comment:
Maybe change it to PREFIX_ITEM so that it follows the class name similar to other fields?

Reply:
Noted, change is made in latest update.